x/mm2/types/market.go

```diff
@@ -57,10 +57,11 @@ func (m *Market) ValidateBasic() error {
 		}
 
 		// check for duplicate providers
-		if _, seen := seenProviders[providerConfig.Name]; seen {
-			return fmt.Errorf("duplicate provider found: %s", providerConfig.Name)
+		key := providerConfig.Name + providerConfig.OffChainTicker
+		if _, seen := seenProviders[key]; seen {
+			return fmt.Errorf("duplicate (provider, off-chain-ticker) found: %s", key)
 		}
-		seenProviders[providerConfig.Name] = struct{}{}
+		seenProviders[key] = struct{}{}
```

Comment:
A given provider might show up twice in a given market, for example BTC-USD and BTC-USDT for coinbase.

Reply:
We should have a separate check that the off-chain-ticker has a unique json field just so that aren't two separate (off-chain, json) fields.